src/lib/RpcApi.ts

```diff
@@ -150,11 +155,24 @@ export default class RpcApi {
                 let account;
                 // Simply testing if the property exists (with `'walletId' in request`) is not enough,
                 // as `undefined` also counts as existing.
-                if (request && (request as ParsedSimpleRequest).walletId) {
-                    account = await WalletStore.Instance.get((request as ParsedSimpleRequest).walletId);
-                    if (!account) {
-                        // no account found although it's required
-                        this.reject(new Error('Account ID not found'));
+                if (request) {
+                    let accountRequired;
+                    if ((request as ParsedSimpleRequest).walletId) {
+                        accountRequired = true;
+                        account = await WalletStore.Instance.get((request as ParsedSimpleRequest).walletId);
+                    } else if (requestType === RequestType.SIGN_TRANSACTION) {
+                        accountRequired = true;
+                        const address = (request as ParsedSignTransactionRequest).sender;
+                        account = this._store.getters.findWalletByAddress(address.toUserFriendlyAddress());
```

Comment:
Typescript sadly does not throw an error here, because `this._store.getters` is `any`, but the `findWalletByAddress()` function expects _two_ arguments, the second being whether to include contracts in the search or not. Right now, you do not, but you must.